prusti-tests/tests/verify/pass/equality/nostd.rs

```diff
@@ -2,8 +2,6 @@
 //! [Prusti: invalid specification] use of impure function "core::cmp::PartialEq::eq" in pure code is not allowed
 //! [Prusti: invalid specification] use of impure function "core::cmp::PartialEq::ne" in pure code is not allowed
 
-#![no_std]
```

Comment:
What about renaming this file to `core_eq.rs`, since we can't test `#![no_std]`?